1/IsArraySorted.js

```diff
@@ -0,0 +1,14 @@
+function isArraySorted (array) {
+  if(array instanceof Array) {
```

Comment:
please try to avoid nested if conditions. Use smth like this: if(!(array instanceof Array)) { return null}